pkg/config/nodetreemodel/config.go

```diff
@@ -916,9 +919,10 @@ func (c *ntmConfig) AllSettingsBySource() map[model.Source]interface{} {
 
 // AllSettingsWithSequenceID returns the settings and the sequence ID.
 func (c *ntmConfig) AllSettingsWithSequenceID() (map[string]interface{}, uint64) {
+	c.maybeRebuild()
```

Comment:
Could you also add a call to `c.maybeRebuild` at the start of `AllSettingsBySource`.